src/h5web/visualizations/shared/Axis.tsx

```diff
@@ -1,58 +0,0 @@
-import React from 'react';
-import { useMeasure } from 'react-use';
-import { AxisLeft, AxisBottom } from '@vx/axis';
-import { format } from 'd3-format';
-import { scaleLinear } from 'd3-scale';
-import styles from './AxisGrid.module.css';
-import { adaptedNumTicks } from './utils';
-import { Domain } from './models';
-
-type Orientation = 'bottom' | 'left';
-
-interface Props {
-  className: string;
-  domain: Domain;
-  orientation: Orientation;
-}
-
-function Axis(props: Props): JSX.Element {
-  const { className, domain, orientation } = props;
-
-  const [min, max] = domain;
-  const isLeftAxis = orientation === 'left';
-  const DirectionalAxis = isLeftAxis ? AxisLeft : AxisBottom;
-
-  const [divRef, { width, height }] = useMeasure();
```

Comment:
You're actually removing two `useMeasure`, nicely done! I don't know why we didn't realise we could have used `useThree` here to get the dimensions... 🤦